pytest_asyncio/plugin.py

```diff
@@ -234,8 +238,15 @@ def _preprocess_async_fixtures(
                         f"instead."
                     )
                 )
-            _inject_fixture_argnames(fixturedef, event_loop_fixture_id)
-            _synchronize_async_fixture(fixturedef, event_loop_fixture_id)
+            # TODO: Fix type errors below.
```

Comment:
The `event_loop_node.stash.get` line can make `event_loop_fixture_id` be `None`, but the functions below don't accept `None`. I'm not sure if:

- This indicates a real problem
- Semantically can't be `None` here (should add an assert)
- The functions should accept `None`

For now I added an ignore which keeps the runtime behavior the same, but let me know if you want another fix.

Reply:
It shouldn't be possible for `event_loop_fixture_id` to be None here.
I removed the type ignore statements in favour of an `assert`.